pkg/services/featuremgmt/toggles_gen.json

```diff
@@ -808,32 +808,17 @@
     {
       "metadata": {
         "name": "dashboardRestore",
-        "resourceVersion": "1719321877748",
+        "resourceVersion": "1728391885555",
         "creationTimestamp": "2024-05-16T17:36:26Z",
         "annotations": {
-          "grafana.app/updatedTimestamp": "2024-06-25 13:24:37.748284 +0000 UTC"
+          "grafana.app/updatedTimestamp": "2024-10-08 12:51:25.555626 +0000 UTC"
         }
       },
       "spec": {
-        "description": "Enables deleted dashboard restore feature (backend only)",
+        "description": "Enables deleted dashboard restore feature",
         "stage": "experimental",
         "codeowner": "@grafana/search-and-storage",
-        "hideFromAdminPage": true
-      }
-    },
-    {
-      "metadata": {
-        "name": "dashboardRestoreUI",
-        "resourceVersion": "1720021873452",
-        "creationTimestamp": "2024-06-25T14:43:13Z",
-        "annotations": {
-          "grafana.app/updatedTimestamp": "2024-07-03 15:51:13.452477 +0000 UTC"
-        }
-      },
-      "spec": {
-        "description": "Enables the frontend to be able to restore a recently deleted dashboard",
-        "stage": "experimental",
-        "codeowner": "@grafana/grafana-frontend-platform",
+        "hideFromAdminPage": true,
         "expression": "false"
       }
```

Comment:
did you manually edit this json to remove this. I _thought_ deleted toggles remained in this file (but with a deletionTimestamp), but i could be wrong

Reply:
I did. I thought it needs to be removed, I didn't mange to remove it by running the related test and didn't know that it can stay there. I'll fix this.

Reply:
Yeah, we really should avoid manually editing generated files.

for toggles, the registry.go file should be the only file that's edited, and then run `make gen-feature-toggles` to updated everything else.